question/app.js

```diff
@@ -0,0 +1,155 @@
+//Q1 変数
+let nickname ='ごっしー';
+let age = 28;
+let greet = '私のニックネームは' + nickname + 'です。' + '年齢は' + age + 'です。';
+console.log(greet);
+
+//Q2 配列
+let languages = ['JavaScript', 'PHP', 'Ruby', 'Python', 'Go'];
+let languagetext = '私の好きな言語は' + languages[0] + 'です。次は' + languages[3] + 'を勉強してみたいです。';
+console.log(languagetext);
+
+//Q3 オブジェクト
+let user = {
+  name: 'John',
+  age: 26,
+  bloodType: 'A',
+  favorite: 'card',
+};
+console.log(user.age);
+
+//Q4 配列×オブジェクト
+let playerList = [
+  {
+    name: 'John',
+    age: 26,
+    favorites: ['Card Game', 'Basket Ball', 'Programming'],
+  },
+  {
+    name: 'Bob',
+    age: 33,
+    favorites: ['Tinder', 'The Legend of Zelda'],
+  },
+  {
+    name: 'Michael',
+    age: 22,
+    favorites: ['Football', 'Smash Bros.'],
+  },
+];
+console.log(playerList[1].favorites[1]);
+
+//Q5 四則演算
+let sum = 0;
+
+for (let i = 0; i < playerList.length; i++) {
+  sum += playerList[i].age;
+  avarage = sum / 3;
+}
+console.log(avarage);
+
+//Q6 関数
+function sayHello() {
+  console.log('Hello');
+}
+sayHello();
+
+let sayWorld = function () {
```

Comment:
[must]
`function`のあとは空白開けないです！
関数を実行するときも開けないですよね。
```suggestion
let sayWorld = function() {
```

Reply:
- 間違えて空白入れてしまいました。修正いたしました。

Reply:
- 修正いたしました。